UnityProject/Assets/Scripts/Util/ThreadSafeList.cs

```diff
@@ -36,6 +36,17 @@ public bool Remove(T item)
 		}
 	}
 
+	public void AddIfMissing(T item)
```

Comment:
is this supposed to be an extension?

Reply:
Ah nevermind, I just realized ThreadSafeList is a custom class, I thought it was part of C# libraries.